deployment/cloud-run/debug_errorhandler_detailed.py

```
@@ -6,74 +6,74 @@
-print(f"Built-in errorhandler: {getattr(__builtins__, 'errorhandler', 'Not found')}")
-print(f"Global errorhandler: {globals().get('errorhandler', 'Not found')}")
+print"\n🔍 Checking for global variable conflicts..."
+print(f"Built-in errorhandler: {getattr__builtins__, 'errorhandler', 'Not found'}")
+print(f"Global errorhandler: {globals().get'errorhandler', 'Not found'}")
 
 # Let's check if there's a version issue
 try:
     import flask_restx
-    print(f"\n🔍 Flask-RESTX version: {flask_restx.__version__}")
-    print(f"Flask version: {flask.__version__}")
+    printf"\n🔍 Flask-RESTX version: {flask_restx.__version__}"
```

Ruff (0.12.2)

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

Trailing whitespace

(W291)